### ISSUE TITLE
Prevent wpa_supplicant to run

### ISSUE THREAD

**Issue by martignoni** [moodlebox/moodlebox#204]
As reported by @jvonau: "`wpa_supplicant` is started anyway by `dhcpcd` via `/lib/dhcpcd/dhcpcd-hooks/10-wpa_supplicant`". See https://github.com/moodlebox/moodlebox/issues/199#issuecomment-823546160.

We should stop this.

**Comment by martignoni** [comment]
@jvonau: so you mean I can let it run without experiencing problems with `hostapd`?

**Comment by jvonau** [comment]
I've messed with this networking stuff at [iiab](https://github.com/iiab/iiab) over the years, in the latest iteration of RpiOS out of the box the WiFi chip is rfkill soft blocked unless you run raspi-config or provide a suitable wpa_supplicant.conf with the country code included placed in the root of the first partition, like touch ssh prior to the booting the new image. The user provided wpa_supplicant.conf is moved into /etc/wpa_supplicant/ then the first-boot code notes that the country code is set and unblocks the chip with rfkill, that is how the headless Zero W wifi stuff is intended to work. Without having any upstream connection info included the country code is set but otherwise no control of the WiFi device is taken and hostapd will take control of the device, just have to make sure the country code is the same in both services and things should just work. I have other reasons for keeping wpa_supplicant operational as I have implemented #77
